File: app/src/main/java/com/app/gamer_shop/models/Order.kt
package com.app.gamer_shop.models

data class Order(
    val id: Int,
    val userId: Int,
    val totalPrice: Float,
    val status: String?,
    val createdAt: String?,
    val country: String?,
    val address: String?,
    val postalCode: String?,
    val firstname: String?,
    val lastname: String?,
    val email: String?
)

File: backend/model/order.go
package model

import "time"

type Order struct {
	UserId     int       `json:"user_id"`
	Country    string    `json:"country"`
	Address    string    `json:"address"`
	PostalCode string    `json:"postal_code"`
	FirstName  string    `json:"firstname"`
	LastName   string    `json:"lastname"`
	Email      string    `json:"email"`
	CreatedAt  time.Time `json:"created_at"`
	TotalPrice float64   `json:"total_price"`
}

type OrderItem struct {
	OrderId     int     `json:"order_id"`
	ProductId   int     `json:"product_id"`
	ProductName string  `json:"product_name"`
	Quantity    int     `json:"quantity"`
	Price       float64 `json:"price"`
}
